### DISCUSSION TITLE
Can I use a key starting with underscore?

### DISCUSSION THREAD

**Discussion by dimitre** [nlohmann/json#4392] [Q&A]
It seems to be a valid JSON using a validator:
```json
{
	"_openFrameworksProjectGeneratorVersion": "45"
}
```
but in some systems while running actions I found a strange issue:
```
Creating project files for osx
terminate called after throwing an instance of 'nlohmann::json_abi_v3_11_2::detail::parse_error'
  what():  [json.exception.parse_error.109] parse error: array index '_openFrameworksProjectGeneratorVersion' is not a number
/home/runner/work/openFrameworks/openFrameworks/scripts/dev/create_package.sh: line 176: 24002 Aborted
```

Is it OK to use a key starting with an underscore?
Thanks

**Comment by gregmarr** [comment]
While it says `parse_error` in the type, it's actually an error in `json_pointer` handling.  You have used `_openFrameworksProjectGeneratorVersion` to try to access an array instead of an object.
